README.md

```diff
@@ -27,10 +34,299 @@ To build all apps and packages, run the following command:
 pnpm run build
 ```
 
+You can also build a specific app or package by running this command:
+
+```bash
+pnpm dlx turbo run build --filter=checkout
+# or
+pnpm run build:checkout
```

Comment:
Is there any difference between those commands? If not, it would be better to stick with one (second one is shorter, more consistent).

Reply:
the first one would allow to run two or more selected apps from the list, but for now we only have 2 :D
so I'll remove the 1st one and stick with `pnpm run build:checkout`